load-testing/load-test-basic-auth-sample.json

```diff
@@ -0,0 +1,139 @@
+{
+  "objective": "Test metadata service with basic auth",
+  "details": {
+    "name": "nci-crdc-staging.datacommons.io"
+  },
+  "service": "metadata-service",
+  "load_test_scenario": "create-and-query",
+  "mds_test_data":  {
+    "filter1": "a=1",
+    "filter2": "nestedData.b=2",
+    "fictitiousRecord1": {
+      "a": 1
+    },
+    "fictitiousRecord2": {
+      "nestedData": {
+        "b": 2
+      }
+    }
+  },
+  "basic_auth": {
+    "username": "<insert_username_here>",
```

Comment:
should we change these to env variables?

Reply:
The JSON is a sample and it is meant to be manually tailored by the user.
There is no mechanism to seamlessly inject an environment variable into the JSON.

We could.however, in future versions, let the user decide whether the secret is hardcoded in there or have something like ENV:MY_VARIABLE and then we can parse the JSON and try to fetch the corresponding variable whenever we find the `ENV:` string. Something like that.

Reply:
sounds good